_pages/en/mandate.md

```diff
@@ -8,40 +8,52 @@ sections: Mission
+<img src="assets/images/our-mission.png" alt="">
+Enable the strategic value of IT within ESDC by reducing its risks and increasing business agility.
 
-## Mission (team's reason for existence)
+<img src="assets/images/our-goals.png" alt="">
+- Develop transformation growth strategies for the organization
+- Help the organization to adopt  DevOps that dramatically reduce risks associated with technology investments and  unlock significant service improvement capabilities
+- Deliver value very quickly, using very short lead times
+- Speed up the time to organization, apply incremental improvements in response to the changing environment, and create a more streamlined development process
+- Continuously iterate and innovate 
 
-Enable the strategic value of IT within ESDC by reducing its risks and increasing business agility.
+<img src="assets/images/our-vision.png" alt="">
+Our vision is to release changes of all kinds on demand quickly, safely, and sustainably so that [Canadians are benefiting seamlessly](human-development-life-cycle.html) of the federal government services through modern technologies.
+Continuous improvement transformation and GC wide "Government as a service" based on [GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html) and the [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601). See [Medium Term IT Picture - 2025](it-picture-medium-term.html) and [Long Term IT Picture - 2030](it-picture-long-term.html).
 
-## Mandate (team's authority within IITB)
 
-- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
-- Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks
-- Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
-- Challenge current thinking and approaches
 
-## Team's Guiding Principles
+### **Team's Guiding Principles**
 
-## Work collaboratively
+- Work collaboratively
 
 Collaborate with anyone and everyone who can _add value_.
 This includes, but is not limited to, all Canadians, levels of government, departments, private sector organizations, and governments.
 
-## Continuous learning
+- Continuous learning
 
 Learning is a core part of daily activities to ensure up to date, modern, and well-researched direction.
 
-## Lead by Example
+- Lead by Example
 
 By using the approaches and technologies we champion, we are able to better respond and anticipate potential challenges.
 We also experiment with modern ways of working in order to demonstrate their value and effectiveness.
 
-## Team understanding of the "digital transformation"
+### **Team understanding of the "digital transformation"**
```

Comment:
Wondering about the rationale for this section? It doesn't seem to fit logically under 'mandate'. And it feels too specific to fit under 'About Us'. Would this content be better as a blog post?

Reply:
This section originally was on the mandate page. I like the idea of making it a blog. Should we remove it from the Mandate page?